mappings/net/minecraft/client/render/entity/model/EntityModelSign.mapping

```diff
@@ -0,0 +1 @@
+CLASS none/blb net/minecraft/client/render/entity/model/EntityModelSign
```

Comment:
I'd say entities and block entities are different enough that it makes sense for their models to be in separate packages.